homeassistant/components/bom/sensor.py

```diff
@@ -73,7 +73,7 @@
     "swell_dir_worded": ["Swell Direction", None],
     "swell_height": ["Swell Height", "m"],
     "swell_period": ["Swell Period", None],
-    "vis_km": ["Visability km", "km"],
+    "vis_km": ["Visibility km", "km"],
```

Comment:
This is a breaking change as it affects the name and entity_id of the corresponding entity.

Reply:
Reverted.